### DISCUSSION TITLE
CPU technical requirements

### DISCUSSION THREAD

**Discussion by vpts1202** [Chia-Network/chia-blockchain#17381] [Support]
Hello
I am newbie in Chia/PoST algo. What is the minimum CPU technical requirements if plotting is via third-party plotting service(i.e. mentioned CPU is not involved ) and plots itself are in separate cloud storage?
Thanks.

**Comment by BrandtH22** [comment]
Hey @vpts1202 , as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

As for hardware - we generally do not recommend farming plots that are stored in the cloud due to higher latency effecting the potential to win rewards and many cloud providers have established rules against plotting/farming in the cloud.

That being said, the recommended hardware for the various plotters can be found here https://docs.chia.net/plotting-hardware/
As for farming one will need to have a fast ssd with likely 250GB + (the current blockchain db is ~140GB and growing) and otherwise something like a raspberry pi can farm standard plots without issues so the CPU, RAM, and IO resources required are fairly low.

**Reply by vpts1202** [reply]
Thank you for your prompt feedback.
- All cloud drawbacks are known, however there is no alternatives to scale.
- Noted with thanks important remark regarding SSD (min 250GB) as I was thinking that at farming stage fast SSD is not really the matter.
- May I ask your advise/recommendation for the other minimal parameters, since for example specs of your mentioned Raspberry pi are also vary(RAM from 256MB to 8GB, similar varies in CPU,etc.)

Look forward to hear from you.
Thanks.

**Reply by BrandtH22** [reply]
Hey @vpts1202 , apart from the minimum required hardware for farming standard plots one would need to look at what they are trying to accomplish with their farm and use hardware that accommodates that. For instance, if you are just planning to use standard plots then something like a raspberry pi will work without issue but if you are planning for a high compression level then you will likely need a gpu and a better cpu. 

Potentially we should approach the question from the opposite side, what size farm are you looking to setup and what compression level of plots?

**Reply by vpts1202** [reply]
@BrandtH22-thank you.
If decision to move in Chia mining(farming) will be taken, then I am planning to use Chia standard plots only. I have no shortage in storage space availability, my issue is computational power, therefore I am here.
Raspberry pi specs vary widely. To test/experimenting from low-end with gradual increases is time consuming, that's why  I am looking for practical advise.
Thanks.

**Reply by BrandtH22** [reply]
Hey @vpts1202 , providing more information about your intended setup will help greatly with providing recommendations. 
Running an 8GB Rpi or similar is sufficient to farm a large number of plots but the bottleneck will be the plotting process as that is resource intensive. 

You may want to join our discord server (https://discord.gg/chia) and ask in the plotting-and-farming channel there 👍

**Reply by vpts1202** [reply]
Your advise for RAM 8GB/SSD 250GB+ for CPU at farming stage, vs chia.net recommended RAM 4GB/SSD 240GB for plotting(!) standard chia plots, is really confusing me:)
Anyway, thank you for your time( at least you have tried to help)
